task/07-yield-tasks.js

```diff
@@ -47,7 +58,19 @@ function* get99BottlesOfBeer() {
  *
  */
 function* getFibonacciSequence() {
-    throw new Error('Not implemented');
+    let fn1 = 1,
+        fn2 = 0;
+    while (true) {
+        let current = fn2;
+        fn2 = fn1;
+        fn1 = fn1 + current;
+        let reset = yield current;
+        if (reset) {
```

Comment:
`reset` functionality is not clear, it doesn't work now, what was the idea?

Reply:
Passing true as the next() argument leads to the restarting the sequence,additional functionality from learning resources